src/background.js

```diff
@@ -357,8 +357,27 @@ const rollout = {
   },
 };
 
+function studyCheck(study) {
+
+  if (study === undefined) {
+    // throw new Error("No study associated with this extension!");
+    return false;
+  }
+
+  const branch = study.branch;
+  switch (branch) {
+  case "doh-rollout-heuristics":
+    return true;
+  case "doh-rollout-disabled":
+    return false;
+  default:
+    throw new Error("Unexpected Study branch: ", branch);
```

Comment:
I don't think the `Error` constructors takes multiple arguments like this. Perhaps this?

```suggestion
    throw new Error(`Unexpected study branch: ${JSON.stringify(branch)}`);
```